src/mudlet.cpp

```diff
@@ -3478,96 +3534,122 @@ void mudlet::slot_module_manager_destroyed()
 
 // Convenience helper - may aide things if we want to put files in a different
 // place...!
-QString mudlet::getMudletPath(const mudletPathType mode, const QString& extra1, const QString& extra2)
+QString mudlet::getMudletPath(const mudletPathType mode, const QString& extra1, const QString& extra2, const QString& homePath)
```

Comment:
I think `profilePath` is more intuitive than `homePath`. Also since it's provided almost everywhere, I'd re-order the arguments to have it first - that would be more intuitive to read too. Let me know if you have that kind of refactoring ability - if no I can use CLion, it allows you to change the order of arguments without a ton of work.